ui/app/helpers/constants/routes.js

```diff
@@ -55,6 +55,64 @@ const SIGNATURE_REQUEST_PATH = '/signature-request'
 const DECRYPT_MESSAGE_REQUEST_PATH = '/decrypt-message-request'
 const ENCRYPTION_PUBLIC_KEY_REQUEST_PATH = '/encryption-public-key-request'
 
+// Used to pull a convenient name for analytics tracking events. The key must
+// be react-router ready path, and can include params such as :id for popup windows
+const PATH_NAME_MAP = {
+  [DEFAULT_ROUTE]: 'Home',
+  [UNLOCK_ROUTE]: 'Unlock Page',
+  [LOCK_ROUTE]: 'Lock Page',
+  [`${ASSET_ROUTE}/:asset`]: `Asset Page`,
+  [SETTINGS_ROUTE]: 'Settings Page',
+  [GENERAL_ROUTE]: 'General Settings Page',
+  [CONNECTIONS_ROUTE]: 'Connections Settings Page',
+  [ADVANCED_ROUTE]: 'Advanced Settings Page',
+  [SECURITY_ROUTE]: 'Security Settings Page',
+  [ABOUT_US_ROUTE]: 'About Us Page',
+  [ALERTS_ROUTE]: 'Alerts Settings Page',
+  [NETWORKS_ROUTE]: 'Network Settings Page',
+  [CONTACT_LIST_ROUTE]: 'Contact List Settings Page',
+  [`${CONTACT_EDIT_ROUTE}/:address`]: 'Edit Contact Settings Page',
```

Comment:
Is this variable name (`address`) used anywhere? :thinking: I noticed that we use `:id` for this route elsewhere, but I don't think it matters, and `address` is more descriptive.

Reply:
oh hmm, this actually is probably worth calling out and maybe preventing -- It is used, in the page track call we pass the params from the matchRoute function which when a user visits `${CONTACT_EDIT_ROUTE}/${my_address}` would send this to segment:

```
params: { address: my_address }
```

this would allow us to identify accounts with metametrics id. This seems like we definitely want to avoid this. Maybe we don't need to send the route params to segment at all? it seemed cool for the asset page where the params would be the token address or ETH

Reply:
Ah, good point, yes we should definitely not send addresses. I agree that the asset seems useful though :thinking: 

I wonder is there a way to send one but not the other?

Reply:
i'm going to change 'account' to 'address' for the my_account routes, and then omit that from params.

Reply:
https://github.com/MetaMask/metamask-extension/pull/9382/commits/f20ab97e0739bdad960bdf81c4a322ff9f9e80da i ended up not changing the param name, but instead omitted both